tests/test_meetings.py

```diff
@@ -13,8 +15,8 @@ def meetings_page(driver):
     return meetings_page
 
 
-@pytest.mark.skipif(settings.STAGE2, reason='No meetings on staging2')
-@pytest.mark.skipif(settings.TEST, reason='Only one meeting on test')
+# @pytest.mark.skipif(settings.STAGE2, reason='No meetings on staging2')
+@pytest.mark.skipif(settings.STAGE1, reason='Only one meeting on test')
```

Comment:
```suggestion
@pytest.mark.skipif(settings.STAGE1, reason='Only one meeting on staging1')
```

Reply:
^ Fixed copy/paste error. 

If we set up the test meetings and submissions properly on all servers, we should actually be ok, and might be able to forego "skipif" all together.